README.md

```diff
@@ -19,6 +19,23 @@ Feel free to [install the add-on][784891c9], read
 [Contributor’s document](CONTRIBUTING.md), if you want to get involved and maybe
 [fix some bugs][7c43e6dd].
 
+#### Incompatibility Notes
+
+We are incompatible with a number of add-ons:
+
+- Hide Caption Titlebar Plus
+- HTitle
+- Tab Mix Plus
+- Tab Groups
+  - Tab Groups Helper
+- Tree-Style Tabs
+- All-In-One Sidebar
+- Firebug
+- Personas Plus
```

Comment:
No, the bug report showed problems with the users lightweight theme (a.k.a. persona), but the actual problem was caused by Hide Caption Titlebar Plus.  Also, how hard would it be to link these to their AMO pages, in case people wanted to check them out as alternatives?